### PR TITLE
Integrate the auto-generate gitignore and fix font

### PR DESCRIPTION
**Pull request by christophechevalier**

### CODE REVIEW

File: resources/style.css [hunk 1]
```diff
@@ -2,15 +2,16 @@
  * Copyright (c) Linagora, 2012-2019
```
Comment (cdeneux) [inline resources/style.css]: Update end date of copyright: 2020

File: resources/font.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
+/* Downloaded from https://fonts.googleapis.com/css?family=Titillium+Web:200,300,400,600,700 */
```
Comment (cdeneux) [inline resources/font.css:5]: Add a file header with copyright and licence

**Review by cdeneux — APPROVED** [review]

File: resources/style.css (hunk 1 above)
Comment (vincent-zurczak) [reply resources/style.css]: +1

File: resources/font.css (hunk 2 above)
Comment (vincent-zurczak) [reply resources/font.css:5]: Indeed, proceed like in the other CSS.

**Review by vincent-zurczak — CHANGES_REQUESTED** [review]
Please, fix the very minor stuff Christophe mentioned.
Use `git --amend` and `git push --force` on your branch to update it.

File: resources/font.css (hunk 2 above)
Comment (christophechevalier) [reply resources/font.css:5]: Done

File: resources/style.css (hunk 1 above)
Comment (christophechevalier) [reply resources/style.css]: Done

**Review by vincent-zurczak — APPROVED** [review]

**Pull request merged**